var/spack/repos/builtin/packages/mvapich2/package.py

```diff
@@ -217,6 +220,12 @@ def configure_args(self):
         else:
             args.append('--enable-fast=all')
 
+        if '+cuda' in self.spec:
+            args.extend([
+                '--enable-cuda',
+                '--with-cuda={0}'.format(spec['cuda'].prefix)
+            ])
```

Comment:
Can you add:
```python
else:
    args.append('--disable-cuda')
```
I don't want it to optionally build CUDA support just because it found it installed on the system.